core/domain/classifier_domain.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2016 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Domain objects for classifier models"""
+
+import copy
+
+class Classifier(object):
+    """Domain object for a classifier."""
+
+    def __init__(self, classifier_id, exp_id, exp_version_when_created,
```

Comment:
Here and elsewhere: please add docstrings, following the example of some other completed files in #2394. Indicate types and meanings of args.